graylog2-server/src/main/java/org/graylog/events/rest/EventDefinitionsResource.java

```diff
@@ -175,7 +176,7 @@ public PageListResponse<EventDefinitionDto> getPage(@ApiParam(name = "page") @Qu
                                                                   allowableValues = "title,description,priority,status")
                                                         @DefaultValue(DEFAULT_SORT_FIELD) @QueryParam("sort") String sort,
                                                         @ApiParam(name = "order", value = "The sort direction", allowableValues = "asc, desc")
-                                                        @DefaultValue(DEFAULT_SORT_DIRECTION) @QueryParam("order") String order) {
+                                                            @DefaultValue(DEFAULT_SORT_DIRECTION) @QueryParam("order") SortOrder order) {
```

Comment:
@bernd what is the proper indentation level here? If I select Code > Reformat Code from the IntelliJ menu I get the line in red. But on save it reverts to the new line in green.

Reply:
@kingzacko1 Yeah, I would say the red line is the correct one. I'm not sure why it's changed on save.

Reply:
Thanks! I'll figure it out and get consistent behavior in my IDE.